### PR TITLE
Revised r2modman profile sharing

### PR DESCRIPTION
**Pull request by Boxofbiscuits97**
## Profile Sharing
Pertains to issue #6  and #27 

Just a cleaned up version to be easily used on the wiki following the style of the other guides.

### CODE REVIEW

**Comment by ghotato** [comment]
thanks a bunch!!

**Comment by ghotato** [comment]
like this way better than mine, sweet and simple.

File: docs/user-guide/installation/installing-r2modman.md
Comment (legoandmars) [inline docs/user-guide/installation/installing-r2modman.md:74]: typo in selection

**Review by legoandmars — CHANGES_REQUESTED** [review]

**Comment by legoandmars** [comment]
I do think this would be better as a separate page, but #29 changes how file structure works, so adding the page would have to be redone anyways

Once #29 is merged I'll probably add a bit more detail and separate this into a new page myself

in the meantime i'll just merge this as-is once the typo is fixed

**Pull request closed without merging**
